WalletWasabi.Backend/Controllers/SoftwareController.cs

```diff
@@ -11,7 +11,12 @@ namespace WalletWasabi.Backend.Controllers
 	[Route("api/[controller]")]
 	public class SoftwareController : Controller
 	{
-		private readonly VersionsResponse VersionsResponse = new VersionsResponse { ClientVersion = Constants.ClientVersion.ToString(3), BackendMajorVersion = Constants.BackendMajorVersion };
+		private readonly VersionsResponse VersionsResponse = new VersionsResponse
+		{
+			ClientVersion = Constants.ClientVersion.ToString(3),
+			BackendMajorVersion = Constants.BackendMajorVersion,
+			LegalDocumentsVersion = Constants.LegalDocumentsVersion.ToString()
```

Comment:
If silent updates are required this needs to be handled differently.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/2805#discussion_r367999891